packages/zosuss/__tests__/__system__/api/Shell.system.test.ts

```diff
@@ -99,4 +100,44 @@ describe("zowe uss issue ssh api call test", () => {
 
     }, TIME_OUT);
 
+    it("should handle errors correctly when connection fails", async () => {
+        const invalidSshSession = new SshSession({
+            hostname: "invalidhost",
+            port: 22,
+            user: "",
+            password: ""
+        });
+
+        try {
+            const command = "uname";
+            let stdoutData = "";
+            await Shell.executeSsh(invalidSshSession, command, (data: string) => {
+                stdoutData += data;
+            });
+        } catch (error) {
+            expect(error.toString()).toContain(ZosUssMessages.unexpected.message);
```

Comment:
Normally for this kind of test, we define an `err` var above the try/catch that we then assign in the catch block to the error thrown, then after the try/catch, we expect `err` to be defined (and any other fields/properties). This way, if no error is thrown we'll fail the test.

Reply:
I've made the change

Reply:
I'm sorry, I should have been more clear and linked an example, what you have is nearly there... the expects should be outside of the catch block:

https://github.com/zowe/zowe-cli/blob/de5afcebddb973d46da4041452202eae866430a4/packages/zosfiles/__tests__/api/methods/download/Download.unit.test.ts#L59

This way, if the test does not throw an error, it'll fail.